testsuite/cluster-test/src/cluster_swarm/job_template.yaml

```diff
@@ -19,11 +19,20 @@ spec:
       containers:
       - name: main
         image: {image}
+        volumeMounts:
+          - mountPath: /opt/libra/data
+            name: data
         imagePullPolicy: Always
         command: ["sh",  "-c", "{command}"]
         securityContext:
+          runAsUser: 0 # To get permissions to write to /opt/libra/data
           privileged: true
           capabilities:
             add: ["SYS_ADMIN"]
       restartPolicy: Never
+      volumes:
+        - name: data
+          hostPath:
+            path: /data
+            type: Directory
```

Comment:
It's probably safer to set type to `DirectoryOrCreate`. Not all hosts will be guaranteed to have the `/data/` directory

Reply:
Yeah, that's actually good idea